Tools/TestWebKitAPI/Tests/WebKitCocoa/ServiceWorkerBasic.mm

```diff
@@ -1750,17 +1750,21 @@ void testSuspendServiceWorkerProcessBasedOnClientProcesses(UseSeparateServiceWor
     [webView2 loadRequest:server.request()];
 
     auto webViewToUpdate = useSeparateServiceWorkerProcess == UseSeparateServiceWorkerProcess::Yes ? webView : webView2;
-    [webViewToUpdate _setThrottleStateForTesting: 1];
-    EXPECT_TRUE(waitUntilEvaluatesToTrue([&] { return ![webViewToUpdate _hasServiceWorkerForegroundActivityForTesting]; }));
-    EXPECT_TRUE(waitUntilEvaluatesToTrue([&] { return [webViewToUpdate _hasServiceWorkerBackgroundActivityForTesting]; }));
 
-    [webViewToUpdate _setThrottleStateForTesting: 2];
```

Comment:
This would previously race `WebProcessProxy::didFinishLaunching` which would reset the state. Placing this inside the `waitUntilEvaluatesToTrue` lambda fixes this race. I think the race started to fail more often because `ProcessThrottler::setThrottleState` wasn't bailing out early as often.